engine/transaction/record/testabilities/mock_repository.go

```diff
@@ -36,7 +35,7 @@ func (m *mockRepository) SaveTX(_ context.Context, txTable *database.Transaction
 	return nil
 }
 
-func (m *mockRepository) GetOutputs(_ context.Context, outpoints iter.Seq[bsv.Outpoint]) ([]*database.Output, error) {
+func (m *MockRepository) GetOutputs(_ context.Context, outpoints iter.Seq[bsv.Outpoint]) ([]*database.Output, error) {
```

Comment:
And the same proposition here (but maybe it's too deep) to have `WillFailOnGetOutputs`? 🤔

Reply:
No problem 👍